vertx-db2-client/src/main/java/examples/DB2ClientExamples.java

```diff
@@ -51,11 +54,28 @@
     
     public static void runJDBC() throws Exception {
         try (Connection con = DriverManager.getConnection("jdbc:db2://" + HOST + ":" + PORT + "/" + DB_NAME, DB_USER, DB_PASS)) {
-            runInitSql(con);
```

Comment:
Changes to this file can be ignored, as it is just for local experimentation (in the future I won't check these in)